common/changes/@microsoft/rush/local-cache-write-disable_2021-11-30-01-27.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/rush",
+      "comment": "Respect the RUSH_BUILD_CACHE_WRITE_ALLOWED environment variable value \"0\" (disabled) even for the local build cache, e.g. to save CPU cycles on CI machines that will throw out the local machine state. Disables cache writes for watch mode commands.",
```

Comment:
Suggested phrasing:

`"(EXPERIMENTAL) Improve the RUSH_BUILD_CACHE_WRITE_ALLOWED environment variable behavior so that it also affects the local build cache. This saves CPU cycles on CI machines that only run a single build. It also avoids cache writes for watch mode commands."`

Reply:
```suggestion
      "comment": "(EXPERIMENTAL) Improve the RUSH_BUILD_CACHE_WRITE_ALLOWED environment variable behavior so that it also affects the local build cache. This saves CPU cycles on CI machines that only run a single build. It also avoids cache writes for watch mode commands.",
```